mkdocs.yml

```diff
@@ -46,40 +68,137 @@ nav:
+      - dist-avg: asl/ref/dist-avg.md
+      - dist-max: asl/ref/dist-max.md
+      - dist-stddev: asl/ref/dist-stddev.md
+      - div: asl/ref/div.md
+      - fadd: asl/ref/fadd.md
+      - fdiv: asl/ref/fdiv.md
       - filter: asl/ref/filter.md
+      - fmul: asl/ref/fmul.md
+      - fsub: asl/ref/fsub.md
       - ge: asl/ref/ge.md
       - gt: asl/ref/gt.md
       - le: asl/ref/le.md
       - lt: asl/ref/lt.md
       - max: asl/ref/max.md
+      - median: asl/ref/median.md
+      - mul: asl/ref/mul.md
       - min: asl/ref/min.md
+      - neg: asl/ref/neg.md
       - or: asl/ref/or.md
+      - pct: asl/ref/pct.md
+      - per-step: asl/ref/per-step.md
+      - percentiles: asl/ref/percentiles.md
+      - pow: asl/ref/pow.md
+      - random: asl/ref/random.md
+      - sqrt: asl/ref/sqrt.md
+      - srandom: asl/ref/srandom.md
       - stat: asl/ref/stat.md
       - stat-avg: asl/ref/stat-avg.md
       - stat-count: asl/ref/stat-count.md
       - stat-last: asl/ref/stat-last.md
       - stat-max: asl/ref/stat-max.md
       - stat-min: asl/ref/stat-min.md
       - stat-total: asl/ref/stat-total.md
+      - stddev: asl/ref/stddev.md
+      - sub: asl/ref/sub.md
+      - sum: asl/ref/sum.md
+      - time: asl/ref/time.md
+      - time-span: asl/ref/time-span.md
       - topk: asl/ref/topk.md
       - topk-others-avg: asl/ref/topk-others-avg.md
       - topk-others-max: asl/ref/topk-others-max.md
       - topk-others-min: asl/ref/topk-others-min.md
       - topk-others-sum: asl/ref/topk-others-sum.md
-      - sum: asl/ref/sum.md
+    - Stack Manipulation:
+      - -rot: asl/ref/-rot.md
+      - 2over: asl/ref/2over.md
+      - call: asl/ref/call.md
+      - clear: asl/ref/clear.md
+      - depth: asl/ref/depth.md
+      - drop: asl/ref/drop.md
+      - dup: asl/ref/dup.md
+      - each: asl/ref/each.md
+      - fcall: asl/ref/fcall.md
+      - freeze: asl/ref/freeze.md
+      - get: asl/ref/get.md
+      - list: asl/ref/list.md
+      - map: asl/ref/map.md
+      - ndrop: asl/ref/ndrop.md
+      - nip: asl/ref/nip.md
+      - nlist: asl/ref/nlist.md
+      - over: asl/ref/over.md
+      - pick: asl/ref/pick.md
+      - roll: asl/ref/roll.md
+      - rot: asl/ref/rot.md
+      - set: asl/ref/set.md
+      - sset: asl/ref/sset.md
+      - swap: asl/ref/swap.md
+      - tuck: asl/ref/tuck.md
+    - Presentation:
+      - alpha: asl/ref/alpha.md
+      - area: asl/ref/area.md
+      - axis: asl/ref/axis.md
+      - color: asl/ref/color.md
+      - decode: asl/ref/decode.md
+      - des-epic-viz: asl/ref/des-epic-viz.md
+      - filter: asl/ref/filter.md
+      - head: asl/ref/head.md
+      - legend: asl/ref/legend.md
+      - limit: asl/ref/limit.md
+      - line: asl/ref/line.md
+      - ls: asl/ref/ls.md
+      - lw: asl/ref/lw.md
+      - palette: asl/ref/palette.md
+      - order: asl/ref/order.md
+      - s: asl/ref/s.md
+      - sort: asl/ref/sort.md
+      - stack: asl/ref/stack.md
+      - vspan: asl/ref/vspan.md
+    - Stateful Functions:
```

Comment:
From a user perspective these are just a category of math operations. It might be more intuitive to just include them in that section.

Reply:
yeah, and I should add notes later about how they work over time instead of "space"